code/obj/item/plants.dm

```diff
@@ -260,6 +265,14 @@
 	brewable = 1
 	brew_result = "tea"
 
+/obj/item/plant/herb/asomna/robust
+	name = "asomna bark"
+	crop_suffix	= " bark"
```

Comment:
You've got another tab in here, looks like. Uploaded an image to actually show what I mean as I guess how it looks depends on your own settings.
![image](https://user-images.githubusercontent.com/62817778/82162567-8350e800-9873-11ea-9ab4-4baf363821f1.png)

Reply:
Will fix now, damn thats annoying

Reply:
Resolved in latest commit.